test/Libraries/Microsoft.Extensions.AI.Tests/ChatCompletion/FunctionInvokingChatClientTests.cs

```diff
@@ -485,7 +485,13 @@ async Task InvokeAsync(Func<Task> work)
                 Assert.Collection(activities,
                     activity => Assert.Equal("chat", activity.DisplayName),
                     activity => Assert.Equal("Func1", activity.DisplayName),
-                    activity => Assert.Equal("chat", activity.DisplayName));
+                    activity => Assert.Equal("chat", activity.DisplayName),
+                    activity => Assert.Equal(nameof(FunctionInvokingChatClient), activity.DisplayName));
+
+                for (int i = 0; i < activities.Count - 1; i++)
+                {
+                    Assert.Same(activities[3], activities[i].Parent);
```

Comment:
```suggestion
                    // Activities are exported in the order of completion, so all except the last are children of the last
                    Assert.Same(activities[activities.Count - 1], activities[i].Parent);
```

Is this suggestion and explanation valid?